### PR TITLE
Update version ORM from 2 to 3 in docs

### PR DESCRIPTION
**Pull request by andreybolonin**

### CODE REVIEW

File: docs/en/index.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Welcome to Doctrine 2 ORM's documentation!
+Welcome to Doctrine 3 ORM's documentation!
```
Comment (greg0ire) [inline docs/en/index.rst]: I think we should remove that number entirely

Comment (derrabus) [reply docs/en/index.rst]: Agreed.

**Comment by derrabus** [comment]
Even if this is just a trivial change, please take a second to write a meaningful PR title. Thank you.

File: docs/en/index.rst (hunk 1 above)
Comment (andreybolonin) [reply docs/en/index.rst]: done

**Review by derrabus — APPROVED** [review]

**Pull request merged**